Solutions/Digital Shadows/Analytic Rules/Digital_Shadows_incident_creation_exclude.yaml

```diff
@@ -4,7 +4,7 @@ description: 'Digital Shadows Analytic rule for generating Microsoft Sentinel in
 severity: Medium
 requiredDataConnectors:
   - connectorId: DigitalShadows
-    dataTypes: 
```

Comment:
irrelevant removal of whitespace 
packaging tool automatically removes this blankspace